### PR TITLE
Set build type for libusrsctp

### PR DESCRIPTION
**Pull request by chertov**
*Issue #, if available:*
by default libusrsctp uses DEBUG build type
https://github.com/sctplab/usrsctp/blob/master/CMakeLists.txt#L37

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by MushMal — DISMISSED** [review]

**Comment by ycyang1229** [comment]
@MushMal Not sure I mentioned this before. I created on PR on usrsctp for the support of mbedtls. https://github.com/sctplab/usrsctp/pull/579. It is merged into main trunk already. Maybe the team want to upgrade the version of usrsctp.

File: CMakeLists.txt [hunk 1]
```diff
@@ -130,7 +130,9 @@ if(BUILD_DEPENDENCIES)
   )
   build_dependency(srtp ${BUILD_ARGS})
 
-  build_dependency(usrsctp)
+  set(BUILD_ARGS
+      -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE})
```
Comment (disa6302) [inline CMakeLists.txt:134]: Thanks @chertov . Can we set a default for this?

Comment (chertov) [reply CMakeLists.txt:140]: @disa6302 ok, i set a default value to RELEASE in https://github.com/chertov/amazon-kinesis-video-streams-webrtc-sdk-c/blob/3850df646ec10e4de305c8d6b0a7f2e0fe8b1e08/CMakeLists.txt#L36

**Review by niyatim23 — CHANGES_REQUESTED** [review]
Hi, some of the builds are failing with `*** buffer overflow detected ***: terminated` 
Please fix this and open a new PR against `develop` to have this merged 
Thanks!

**Comment by niyatim23** [comment]
Closing since this PR fails checks and is stale. Please edit this PR to fix the builds, change the base branch to `develop` and reopen this

**Pull request closed without merging**